application/libraries/Librivox_search.php

```diff
@@ -85,8 +85,8 @@ function advanced_title_search($params)
 
 			$language_clause = ' AND ( p.language_id = ' . $recorded_language . ' OR p.id IN (' . implode(',', $section_project_ids) . ') ) ';
 			$section_language_clause = '
-				AND ( p.language_id = ' . $recorded_language . '
-					OR st.section_language_id = ' . $recorded_language . ') ';
+				AND ( p.language_id <> ' . $recorded_language . '
+					AND st.section_language_id = ' . $recorded_language . ') ';
```

Comment:
```suggestion
				AND st.section_language_id = ' . $recorded_language;
```
If I'm holding all of the pieces right in my head, I think we can do away with this first clause.  It would seem to discard all sections of a collection project, where the entire collection is in the language we actually want.

I'm new to the GitHub code review process, particularly from this end.  I can always open a new PR, or I can test this one again if you'd like to change it.

Reply:
So apparently I can commit this suggestion (still new to the less frequently used functions of the PR workflow myself), but I'd rather not have two commits for a really simple change. Which I guess means you get to open a new PR (crediting @rillig of course).

Reply:
Sounds good!  I'll file a new PR, most likely tomorrow evening.  :+1:

Reply:
Meh, I have not been holding the pieces correctly in my head.  My suggested change leaves both the Collection projects, _and_ all of their constituent Sections, in the results.  I'm in this deep, so I'll likely still come back with a PR soon, it just won't be this one.

I have a lot of respect for folks who can do this for a living.